### ISSUE TITLE
Manual Mode setting status

### ISSUE THREAD

**Issue by hankys** [hvorragend/ha-blueprints#125]
Hello,
after last update (2024.08.03) the manual override feature seems to work.

But i have a new issue regarding the status helper:
The automation sets helper status "manual" to "true" after automatic opening in the morning on random shutters.
This causes my shading automation to stop, because i set a manual override "Ignore/override next automatic sun shading after manual position changes" on those windows.

**Comment by fisch55** [comment]
same here...

**Comment by hvorragend** [comment]
But isn't that exactly the intention of the feature? Or why have you activated it?

Normally I would say that you should deactivate it. 

Or what am I not understanding here?

**Comment by hankys** [comment]
The problem I am facing is the following:
- my automation is set to automatically open the shutters in the morning when the light intensity is 400lx or more. This works without any problems.
- However, the automation sets the status "manual":{"a":true,....." in the "Cover Status Helper" after opening the shutters, even though the opening was carried out automatically. This affects different shutters. Yesterday there were some different shutters than today.
- By setting the "Cover Status Helper" to the status "manual":{"a":true,....." my automatic sun shading no longer works because I have set the option "Ignore/override next automatic sun shading after manual position changes" for all shutters.

In short: in my case the automation deactivates itself with regard to sun shading.

This problem has only appeared since the last update.

**Comment by fisch55** [comment]
Exactly the Same . Good described!!!

**Comment by hvorragend** [comment]
> * However, the automation sets the status "manual":{"a":true,....." in the "Cover Status Helper" after opening the shutters, even though the opening was carried out automatically. This affects different shutters. Yesterday there were some different shutters than today.

You should adjust the DriveTime values. Probably increase a little. Then roller blind movements triggered by CCA are also assigned to the automation. 

Otherwise show me a trace in which the "Manual" is set incorrectly. But I think it is due to the travel time or the latency of your hardware.  

> This problem has only appeared since the last update.

This is probably because there were still bugs with the override until then. Now it works for the first time.

**Comment by fisch55** [comment]
My Hlepers of there shutters does not works anymore. The states are wrong... Deleted  Helpers and reinstall it, got the old states...
Any Idea, what the Problem is?

<img width="204" alt="Bildschirmfoto 2024-08-06 um 22 07 29" src="https://github.com/user-attachments/assets/3779dffb-b7c8-4d0d-a4dc-abe6fed71c9c">

[trace automation.cover_control_spulenfenster 2024-08-06T20_00_00.636102+00_00.json](https://github.com/user-attachments/files/16515246/trace.automation.cover_control_spulenfenster.2024-08-06T20_00_00.636102%2B00_00.json)

<img width="1148" alt="Bildschirmfoto 2024-08-06 um 22 08 47" src="https://github.com/user-attachments/assets/ccea56d4-e4b4-41fb-87a8-d2e024531e2d">

**Comment by hvorragend** [comment]
This is crazy. The user @Bostil has the same problem. Some of his CCA blueprints do not execute the entire **action** part. The traces are almost empty. Only approx. 37 KB instead of almost 200 KByte. 

Deactivate your automation and reactivate it afterwards.  

https://community.home-assistant.io/t/automation-triggered-but-does-not-run-actions-since-last-update-2022-10-4/476854

**Comment by fisch55** [comment]
Thx for reply - but it does Not help.

That is really Crazy . May it be an issue with ha Beta 2024.08?

**Comment by hvorragend** [comment]
I've been looking for a long time, but look what I've found. 

https://github.com/home-assistant/core/issues/123082

**Comment by bostil** [comment]
> This is crazy. The user @bostil has the same problem.

You have me confused with someone else. I don't know anything about this repo.

**Comment by fisch55** [comment]
> I've been looking for a long time, but look what I've found.
> 
> [](https://github.com/home-assistant/core/issues/123082)

Means it is an Issue of ha Core ….???

**Comment by hankys** [comment]
> You should adjust the DriveTime values. Probably increase a little. Then roller blind movements triggered by CCA are also assigned to the automation.
> 
> Otherwise show me a trace in which the "Manual" is set incorrectly. But I think it is due to the travel time or the latency of your hardware.

That helped !

Yesterday evening i put 1 second more on every shutter. This morning they all stayed in "manual":{"a":false,....." after opening by the automation.

I hope the behavior remains similar for the next few days :)


Thank you very much.

**Comment by hvorragend** [comment]
> > I've been looking for a long time, but look what I've found.
> 
> Means it is an Issue of ha Core ….???

Yes. And we should use a seperate issue for this next time. Offtopic here. 😀

**Comment by hvorragend** [comment]
> That helped !
> 
> Yesterday evening i put 1 second more on every shutter. This morning they all stayed in "manual":{"a":false,....." after opening by the automation.
> 
> I hope the behavior remains similar for the next few days :)
> 
> Thank you very much.

Great. Feel free to round up. It doesn't have to be that exact 

Then I'll close this ticket, okay?

**Comment by fisch55** [comment]
> > > I've been looking for a long time, but look what I've found.
> > 
> > 
> > Means it is an Issue of ha Core ….???
> 
> Yes. And we should use a seperate issue for this next time. Offtopic here. 😀

Ok , sorry…but should I Open One ?

**Comment by hvorragend** [comment]
No, you should update your core beta or wait for the final. And I hope that should solve it.
